nifi-commons/nifi-record/pom.xml

```diff
@@ -27,4 +27,11 @@
         several interfaces for interacting with Records. This module should not depend
         on any external libraries.
     </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+    </dependencies>
```

Comment:
As noted in the above module description, the `nifi-record` module should not have any external dependencies.